lib/autobuild/import/git.rb

```diff
@@ -293,6 +293,52 @@ def validate_importdir(package)
             git_dir(package, true)
         end
 
+        
+        # Return default local branch if exists, if not return nil
+        #
+        # @param [Package] package
+        def default_local_branch(package)
```

Comment:
This name is misleading. It does not returnthe default local branch, it returns the default remote branch as stored locally.

Maybe `try_resolve_remote_head_from_local_remotes`. `default_remote_branch` would become `try_resolve_remote_head_from_ls_remote`. Finally `default_branch` would become `resolve_remote_head`